quipucords/api/scanjob/model.py

```diff
@@ -328,7 +332,10 @@ def queue(self):  # noqa: C901
             for source in self.sources.all():
                 source.most_recent_connect_scan = self
                 source.save()
-
+        # Create report
+        # TODO(?): while Deployments/Details report are not removed, replace their
+        # report_id with report.id (?)
```

Comment:
Are we migrating Deployment/Details reports to the new Report?  Or do we not care (i.e. just zap them) since we're essentially ephemeral until DiscoPop time?

Reply:
we will 🪓 them when the time comes =D